pkg/nsx/services/subnetport/builder.go

```diff
@@ -49,7 +49,7 @@ func (service *SubnetPortService) buildSubnetPort(obj *v1alpha1.SubnetPort, nsxS
 			// AppId:             common.String("nsx.ns-3.pod-5%"),
 			// ContextId:         common.String("95ccaad4-0dfb-469e-a1e6-27d815826382"),
 			Id:         common.String(nsxCIFID.String()),
-			TrafficTag: nil,
+			TrafficTag: common.Int64(0),
```

Comment:
Why cannot it be nil?

Reply:
If we write nil, the created NSX resource's will be 0 automatically. Then when we compare the existing resource (get via NSX API) and the desired resource, the result is not equal (0 != nil). So I set it to 0 to avoid this kind of inequality and avoid a duplicate patch request.